articles/architecture-scenarios/application/mobile-api.md

```diff
@@ -9,30 +9,369 @@ beta: true
 
 # Mobile + API
 
+In this scenario we will build a Timesheet API for a fictitious company named ABC Inc. The API will allow management of timesheet entries for an employee or a contractor.
+
+We will also be building a Mobile Application which will be used to view and log timesheet entries in the centralized timesheet database using the API.
+
+## The Premise
+
+ABC Inc. is a consulting startup company. Currently they have approximately 100 employees and they also outsource several activities to external contractors. All employees and external contractors are required to fill in their timesheets every week. 
+
+The company has built a timesheets application, a scenario we covered in [Single Sign-On for Regular Web Apps](/architecture-scenarios/application/web-app-sso). The internal employees use this web app to fill in their timesheets but the company wants to replace it with a SPA. The app will be used to log timesheet entries and send the data to the centralized timesheet database using the API.
```

Comment:
This paragraph refers to the SPA, we should talk about the mobile app scenario

cc @jerriep